src/catalog/models/udf_io.py

```diff
@@ -103,3 +103,12 @@ def __str__(self):
         column_str += ")\n"
 
         return column_str
+
+    def __eq__(self, other):
```

Comment:
Does python generate a copy constructor by default?

Reply:
I don't think so. In python assignment works as a binding of the target with the original object.